src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +60,138 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
 Refer to [Program.cs](../../examples/runtime-instrumentation/Program.cs) for a
 complete demo.
 
+## Metrics
+
+### GC related metrics
+
+| Name                                            | Description                                                          | Units     | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------------------------------|----------------------------------------------------------------------|-----------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**gc.collections.count** | Number of times garbage collection has occurred since process start. | `{times}` | ObservableCounter | `Int64`    | gen              | gen0, gen1, gen2 |
```

Comment:
```suggestion
| process.runtime.dotnet.**gc.collections.count** | Number of garbage collections that have occurred since process start. | `{collections}` | ObservableCounter | `Int64`    | generation              | gen0, gen1, gen2 |
```

I notice some units have curly braces around them and others don't, is there a meaning to that?

Reply:
Yes. It is the semantic convention for instruments that measure an integer count of something: https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-units